packages/contracts-bedrock/package.json

```diff
@@ -11,25 +11,29 @@
   "scripts": {
     "bindings": "pnpm bindings:ts && pnpm bindings:go",
     "bindings:ts": "nx generate @eth-optimism/contracts-ts",
-    "bindings:go": "cd ../../op-bindings && make",
-    "build": "forge build",
+    "bindings:go:no-build": "cd ../../op-bindings && make binding-build",
+    "bindings:go": "pnpm clean && pnpm build && bindings:go:no-build",
```

Comment:
`bindings:go` forces a rebuild both before and after this change, but `bindings:go:no-build` provides a much faster option if you don't need to rebuild.